about.html

```diff
@@ -0,0 +1,713 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="BINABOX is the best platform to check the rarity of NFT collection"
+    />
+    <title>About BINABOX</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header>
+      <a href="./index.html" class="logo-block">
+        <img src="./img/img-logo/logo.svg" alt="Binabox logo" />
+        BINABOX
+      </a>
+      <!-- Navigation Menu -->
+      <nav class="nav-menu">
+        <ul class="nav-menu-items">
+          <li class="nav-menu-item">
+            <a href="index.html">HOME</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./about.html">ABOUT</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./index.html#roadmap-block">ROADMAP</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./catalog.html">COLLECTION</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="./faqs.html">FAQS</a>
+          </li>
+
+          <li class="nav-menu-item">
+            <a href="#" class="dropdown">PAGES</a>
+            <ul class="dropdown-content">
+              <li>
+                <a href="./catalog.html">NFT Catalog</a>
+              </li>
+              <li>
+                <a href="./blog.html">Blog</a>
+              </li>
+              <li>
+                <a href="./contact-us.html">Contact Us</a>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="btn-box">
```

Comment:
> nav purpose is to provide navigation links, either within the current document or to other documentsto provide navigation links, either within the current document or to other documents

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

so here is not a nav